boards/arm/nucleo_l496zg/nucleo_l496zg_defconfig

```diff
@@ -0,0 +1,41 @@
+CONFIG_ARM=y
+CONFIG_SOC_SERIES_STM32L4X=y
+CONFIG_SOC_STM32L496XG=y
+CONFIG_CORTEX_M_SYSTICK=y
+# 80MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=80000000
+
+# enable uart driver
+CONFIG_SERIAL=n
```

Comment:
Comment contradicts the defconfig. Remove? But it would be good to have at least the console to start with.